src/material/icon/icon-registry.ts

```diff
@@ -146,10 +146,16 @@ export class MatIconRegistry implements OnDestroy {
 
   /**
    * The CSS classes to apply when an `<mat-icon>` component has no icon name, url, or font
-   * specified. The default 'material-icons' value assumes that the material icon font has been
-   * loaded as described at https://google.github.io/material-design-icons/#icon-font-for-the-web
+   * specified. The default 'material-symbols-outlined' value assumes that the Material Symbols icon font has been
+   * loaded as described at https://fonts.google.com/icons.
+   * The 'material-icons' class is included for backwards compatibility with apps loading
+   * the old Material Icons font set. This default CSS class will be removed in v20.
    */
-  private _defaultFontSetClass = ['material-icons', 'mat-ligature-font'];
+  private _defaultFontSetClass = [
+    'material-symbols-outlined',
```

Comment:
I was under the impression that symbols are a completely different set of icons. Won't this apply both the non-outlined and outlined icon at the same time?

Reply:
Depends on how their index.html is set up. If they are loading both fonts, then there can be a conflict (and I think undefined behavior about which one wins).

Ideally folks are either loading Material Icons or Material Symbols Outlined.

I'm open to ideas on other ways we can solve this. My goal is to avoid doing any LSCs/schematics (but not a requirement)

Reply:
Could we update the docs to tell users that they need to change the URL like you did here and call `iconRegistry.setDefaultFontSetClass('material-symbols-outlined');`?

Reply:
Yeah that's one option - just tell folks how they can set themselves up for Material's latest font set. Would be nice if it was set as the default though. Not sure there's any other way to go about it without it being breaking